src/Pages/Home-Page/Home.jsx

```diff
@@ -4,12 +4,24 @@ import "./responsive.css";
 import { Nav } from "../../components/Nav/Nav";
 import { NavForMobile } from "../../components/NavForMobile/NavForMobile.js";
 import { Footer } from "../../components/Footer/Footer";
+import { HomeScreen } from "./Components/HomeScreen";
+import { Category } from "./Components/Category";
+import { Offer } from "./Components/Offer";
+import { Product } from "./Components/Product";
+import { Services } from "./Components/Services";
 
 export function Home() {
   return (
     <div className="home-page">
       <Nav />
       <NavForMobile />
+      <main>
+        <HomeScreen />
+        <Category />
+        <Offer />
+        <Product />
+        <Services />
+      </main>
```

Comment:
This is good, dividing into small components.

Reply:
ThankYou